src/translations/ru.json

```
@@ -0,0 +1,39 @@
+{
+  "log_in": "Войти",
+  "log_out": "Выйти",
+  "logged_in_as": "Вы вошли как",
+  "logged_in": "Вы вышли",
+  "logging_in": "Вхожу",
```

It would probably make the message sound more formal if it was indefinite.

```suggestion
  "logging_in": "Выполняется вход",
```